crates/rpc-types/src/eth/block.rs

```diff
@@ -25,8 +25,10 @@ pub struct Block {
     pub total_difficulty: Option<U256>,
     /// Uncles' hashes.
     pub uncles: Vec<B256>,
-    /// Transactions.
+    /// Block Transactions. In the case of an uncle block, this field is not included in RPC
+    /// responses, and when deserialized, it will be set to [BlockTransactions::Uncle].
     #[serde(skip_serializing_if = "BlockTransactions::is_uncle")]
+    #[serde(default = "BlockTransactions::uncle")]
```

Comment:
nit merge those two

Reply:
done in https://github.com/alloy-rs/alloy/pull/98/commits/0e1eff8a633fa675265cca80e7a56397ad3ee025